From 89efa5eb515e6c9e0dec52e9a223adb87c78dfc8 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 13 Oct 2014 11:10:27 +0800
Subject: [PATCH 0326/5242] MLK-9684-1: mfd: si476x-i2c: add of_compatible for
 si476x-codec

commit  cad9db1f61b5f2c38e0eb698dcff72d9ba7029a2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add of_compatible for si476x-codec, then si476x-codec driver will have
codec_of_node, So machine driver can use the codec_of_node.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
(cherry picked from commit e2ec44f91a21b127e155e8317d06e8ead7fd2678)
(cherry picked from commit ac6decaf5414e784ae81a524edc2f32060061b59)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mfd/si476x-i2c.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mfd/si476x-i2c.c b/drivers/mfd/si476x-i2c.c
index 3b45a61..48dc07a 100644
--- a/drivers/mfd/si476x-i2c.c
+++ b/drivers/mfd/si476x-i2c.c
@@ -830,6 +830,7 @@ static int si476x_core_probe(struct i2c_client *client,
 	    core->pinmux.xout == SI476X_XOUT_TRISTATE) {
 		cell = &core->cells[SI476X_CODEC_CELL];
 		cell->name          = "si476x-codec";
+		cell->of_compatible = "si476x-codec";
 		cell_num++;
 	}
 #endif
-- 
1.7.9.5

